sklearn_numba_dpex/kmeans/drivers.py

```diff
@@ -313,14 +315,55 @@ def lloyd(
         # refers. For this reason, this strategy is not compatible with sklearn
         # unit tests, that consider new_centroids_t (the array after the update)
         # to be the best centroids at each iteration.
-
         centroids_t, new_centroids_t = (new_centroids_t, centroids_t)
 
+        # ???: if two successive assignations have been computed equal, it's called
+        # "strict convergence" and means that the algorithm has converged and can't get
```

Comment:
What is the meaning of the `???` here? Is it a question?

Reply:
it opens the discsussion regarding diverging from sklearn behavior or considering changing it upstream (https://github.com/scikit-learn/scikit-learn/issues/25716)